DevExtreme.AspNet.TagHelpers.Generator/PropTypeInfo.cs

```diff
@@ -19,15 +20,17 @@ const string
             SPECIAL_RAW_STRING = "sp_raw_string",
             SPECIAL_DOM_TEMPLATE = "sp_dom_template";
 
-        static readonly IDictionary<string, string> _overrideTable = InitOverrideTable();
+        static readonly IDictionary<string, string> _predefinedTypes = InitPredefinedTypes();
 
         public readonly string ClrType;
         public readonly bool IsDomTemplate;
         public readonly bool IsRawString;
 
-        public PropTypeInfo(string tagFullKey, string propName, string rawType) {
+        public PropTypeInfo(XElement element, string fullName, string propName) {
+            var rawType = element.GetRawType();
             var dirtyType =
-                TryGetTypeOverride(tagFullKey + "." + propName, isArray: rawType == "array") ??
+                TryGetPredefinedType(fullName) ??
```

Comment:
Form this point of view these methods are to be named just `GetSmth` or `DoSmth` (without `Try` prefix).
How do you like it, Aleksey?